### ISSUE TITLE
Suggestion: add when-green-flag to default project

### ISSUE THREAD

**Issue by boazsender** [LLK/scratch-gui#2737]
### Expected Behavior
As a new scratcher, I always forget to add a when-green-flag. It would be cool if the default project started with this block on the stage.

**Comment by kyleplo** [comment]
Yeah, I like this idea! Many of my early projects had no green flag block, some, you had to manually drag each sprite into its starting position to use the project again!

**Comment by Naohiro2g** [comment]
Sorry but I disagree with this idea and even I've been telling young learners not place the when-green-flag-pressed block as a habit.

Let's think about the reason why Scratch 1, 2, and Scratch Jr. didn't have the block for every new sprite including the Scratch cat.

There are many event triggers to start something in Scratch world and the green flag is just one of them. It can have the meaning of starting the project but not necessary.

I believe making the block more special is not good for learning how every sprites react the messages or events, sometimes.  The green flag is already a special one and it's enough.
